lib/vehicle.js

```diff
@@ -45,12 +45,15 @@ const METHODS_MAP = {
  * @param {String} [options.unitSystem=metric] - The unit system to use for vehicle data
  * must be either `metric` or `imperial`.
  * @param {Object} [options.version] - API version to use
+ * @param {Object} [options.flags] - Feature Flags (early access)
+
  */
 function Vehicle(id, token, options = {}) {
   this.id = id;
   this.token = token;
   this.unitSystem = options.unitSystem || 'metric';
   this.version = options.version || config.version;
+  this.flags = options.flags || {};
```

Comment:
Need to update `this.request` to send the flags as query parameters 

Make sure to serialize the flags into a string before you send them (search around in the code base to see how this can be achieved)